docs/server/source/quickstart.md

```diff
@@ -57,6 +57,13 @@ I. Run BigchainDB Server:
 $ bigchaindb start
 ```
 
+J. Verify BigchainDB Server setup by visiting the BigchainDB Root Url in your browser (if not changed during installation):
+```text
+$ http://127.0.0.1:9984/
```

Comment:
I think the 127.0.0.1 address might vary but since we're assuming Ubuntu 16.04 or similar, this should be okay?